backend/app/alembic/versions/2026_08_20_0001-a1b2c3d4e5f6_add_married_relationship_type.py

```
@@ -0,0 +1,34 @@
+
+def upgrade() -> None:
+    # The Python RelationshipTypeEnum defines MARRIED but the PostgreSQL
+    # relationshiptypeenum type was created without it. Without this migration,
+    # writing a MARRIED relationship raises InvalidTextRepresentationError:
+    #   invalid input value for enum relationshiptypeenum: "MARRIED"
+    # See AGENTS.md "DB Enums & Alembic Migrations" (manual ALTER TYPE required;
+    # autogenerate does not detect enum value changes).
+    op.execute("ALTER TYPE relationshiptypeenum ADD VALUE 'MARRIED'")
+
+
+def downgrade() -> None:
```

Ruff (0.16.1)

[warning] Missing docstring in public function

(D103)